### DISCUSSION TITLE
How can we extract specific cells more precisely?

### DISCUSSION THREAD

**Discussion by dahun73** [satijalab/seurat#6418] [Q&A] [answered]
Hello everyone :)

In advance, thank you for serving such a nice tool for scRNA-seq analysis. 
Actually, I'm trying to extract excitatory or inhibitory neurons in mouse brain sample. Generally, we can use `FindMarkers` to identify cell types. Then, we can use `subset` function to extract specific cell types. After that, I tried to perform clustering again to find specific cell types. 

However, I realized that there are some unwanted cells. For example, I extracted excitatory neurons but there are a few inhibitory neurons in extracted cells. So, I want to extract some specific cells more precisely. Are there any tips for this?

Thanks

Dahun

**Comment by f6v** [comment, marked as answer]
You can always filter the cells based on the marker expression without relying on clustering. What I do is visualise the expression using the violin plots from `Seurat` to find a threshold. In my case, this could be all cells having CD79A expression value higher than, say, 1.5. You can also use [FeatureScatter](https://satijalab.org/seurat/reference/FeatureScatter.html). But you've got to know a gene for this. You can also try transferring the labels from some other annotated dataset, thus bypassing clustering. 

You might also want to experiment with the number of PCs you select for the analysis. More or fewer PCs can change the clustering. As well as regressing technical effects or removing certain genes from the HVG list.

**Reply by dahun73** [reply]
@f6v 
Thank you for kind explanation! I think I should try your suggestion!
